Bixal/move.mil
# Vuln fix

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

- [ x] run the application locally (`make up`) and verified that my changes behave as expected.
      against my changes
- [ x] summarized below my changes and noted which issues (if any) this pull request fixes or addresses.
- [ x] thoroughly outlined below the steps necessary to test my changes.
- [ x] read, understand, and agree to the terms described in [CONTRIBUTING.md](https://github.com/Bixal/move.mil/blob/master/CONTRIBUTING.md).

## Summary of Changes

This pull request…

- Uninstalls gulp-cssnano (deprecated)
- Uninstalling `gulp-cssnano` gets rid of 1 high and 1 moderate vulnerability on `npm audit`
- Installs gulp-clean-css to replace gulp-cssnano
- Modifies `config/sass.js` to run tasks with gulp-clean-css smoothly

## Testing

To verify the changes proposed in this pull request…

1. Run `npm install`
1. Run `npm audit`
1. Sass functionality running as expected

## Screenshots

_Attach relevant before and after screenshots here._
